package.json

```diff
@@ -48,5 +52,17 @@
   "peerDependencies": {
     "markdown-it": ">=7.0.1"
   },
-  "tonicExampleFilename": "demo.js"
+  "tonicExampleFilename": "demo.js",
+  "browserify": {
```

Comment:
You're right, its not needed since I added the transform param directly to browser script.  I'll remove.